website/docs/tutorials/01-introduction.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_label: Introduction (1 min)
+---
+
+# Introduction
+
+Protostar is a StarkNet smart contract development [toolchain](https://en.wikipedia.org/wiki/Toolchain), which helps you with the following tasks:
+
+- [dependencies management](/docs/tutorials/Guides/dependencies-management)
+- project compilation
+- testing your contract
+
+:::note
+Protostar takes its inspiration from [Foundry](https://onbjerg.github.io/foundry-book/index.html).
+:::
+
+## Target audience
+
+You have basic knowledge about [Cairo and StarkNet](https://www.cairo-lang.org/docs/).
+
+## Learning objectives
+
+After reading this guides, you will know how to do the following:
+
+- install and upgrade Protostar
+- initialize new Protostar project
+- configure `protostar.toml`
+- adapt existing Cairo project to Protostar project
+- compile project
+- [add, remove, and remove dependencies](/docs/tutorials/Guides/dependencies-management)
+- test contracts with the help of cheatcodes
```

Comment:
I would replace it with a table of contents with good section names. Serves the same purpose but would be more useful for a reader. Would make docs easier to use as a reference not a book which you read from the beginning to the end.

Reply:
There's already a sidebar on the left side and document summary on the right.  I am not sure whether adding such TOC is possible in docusaurous.  TOC lists everything, learning objectives focuses on the most important skills.

Also, if that would be the case, Google would replace "Learning objectives" with TOC
https://developers.google.com/tech-writing/one